dotnet/Tiltfile

```diff
@@ -0,0 +1,6 @@
+docker_compose("./docker-compose.yml")
+
+local_resource(
+  'load messages into redis',
+  cmd='docker compose exec redis redis-cli -n 0 SADD messages "how are you?" "how are you doing?" "what\'s good?" "what\'s up?" "how do you do?" "sup?" "good day to you" "how are things?" "howzit?" "woohoo"',
+  resource_deps=['redis'])
```

Comment:
Fancy pant Tilting to populate Redis with the set of messages for the Message service to retrieve.